Lib/lib2to3/tests/test_fixers.py

```diff
@@ -1815,6 +1815,23 @@ class Test_imports2(FixerTestCase, ImportsFixerTests):
     from ..fixes.fix_imports2 import MAPPING as modules
 
 
+class Test_imports3(FixerTestCase):
+
+    def setUp(self):
+        super(Test_imports3, self).setUp(['imports3', 'renames'])
```

Comment:
Can you only include `imports3` in this test class? You can make a separate test class for the combination of `imports3` and `renames`.